src/smac_planner_hybrid.cpp

```diff
@@ -351,13 +348,15 @@ uint32_t SmacPlannerHybrid::makePlan(
 #endif
 
   // Smooth output_path
-  if (_config.smooth_path && num_iterations > 1) {
-    _path_smoother.smooth(output_path, costmap, time_remaining);
-
+  if (_config.smooth_path) {
```

Comment:
Q: is this condition supposed to be removed: `num_iterations > 1` ?

Reply:
yes, I think it becomes true when we we replan to the same goal, so I need to remove it (and is useless anyway)